src/main/java/com/iota/iri/IRI.java

```diff
@@ -143,7 +143,6 @@ private static void validateParams(final Configuration configuration, final Stri
             configuration.put(DefaultConfSettings.COORDINATOR, Configuration.TESTNET_COORDINATOR_ADDRESS);
             configuration.put(DefaultConfSettings.SNAPSHOT_FILE, Configuration.TESTNET_SNAPSHOT_FILE);
             configuration.put(DefaultConfSettings.MILESTONE_START_INDEX, Configuration.TESTNET_MILESTONE_START_INDEX);
-            configuration.put(DefaultConfSettings.SNAPSHOT_SIGNATURE_FILE, Configuration.TESTNET_SNAPSHOT_SIG_FILE);
```

Comment:
I understand the reasoning here - that `snapshotTestnet.sig` is not validated so there is no need to point to it,
However, his means that `snapshotMainnet.sig` will be read also for testnet, even if the signature is not validated it still mixes file names.

better to set as `null` / `""`

Reply:
set to "".
I think `null` will raise an exception when inserted as a value to `ConcurrentHashMap<>`